index.bs

```diff
@@ -1195,7 +1195,7 @@ Advisory Board and Technical Architecture Group Elections</h5>
 	are changes of affiliation).
 	(Other formal relationships such as other contracts should be disclosed as potential conflicts of interest.)
 	Each nomination <em class="rfc2119">should</em> include
-	a few informative paragraphs about the nominee.
+	a few non-normative paragraphs about the nominee.
```

Comment:
I think in this instance, "informative" means "carrying information", not "non-normative", so the replacement is not appropriate.